<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>arguwindow</class>
 <widget class="QWidget" name="arguwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>484</width>
    <height>362</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="func_name_label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>func</string>
   </property>
  </widget>
  <widget class="QScrollArea" name="scroll_area">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>50</y>
     <width>441</width>
     <height>231</height>
    </rect>
   </property>
   <property name="widgetResizable">
    <bool>true</bool>
   </property>
   <widget class="QWidget" name="scrollAreaWidgetContents">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>439</width>
      <height>229</height>
     </rect>
    </property>
    <widget class="QWidget" name="formLayoutWidget">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>10</y>
       <width>421</width>
       <height>211</height>
      </rect>
     </property>
     <layout class="QFormLayout" name="form_layout"/>
    </widget>
   </widget>
  </widget>
  <widget class="QPushButton" name="correct_btn">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>300</y>
     <width>111</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string>确定</string>
   </property>
  </widget>
  <widget class="QLabel" name="argu_count_label">
   <property name="geometry">
    <rect>
     <x>358</x>
     <y>20</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>argu</string>
   </property>
  </widget>
  <zorder>scroll_area</zorder>
  <zorder>func_name_label</zorder>
  <zorder>correct_btn</zorder>
  <zorder>argu_count_label</zorder>
 </widget>
 <resources/>
 <connections/>
</ui>
